.github/mergify.yml

```diff
@@ -65,3 +85,7 @@ pull_request_rules:
       request_reviews:
         users_from_teams:
           - "@csi-addons/maintainers"
+  - name: refactored queue action rule
```

Comment:
This rule really confuses me. It might be correct to try-and-queue everything, and remove things from the queue if `queue_conditions` are not met. But I think it could be written in an easier to understand way (or with extra comments).